doc/extdev/appapi.rst

```diff
@@ -676,8 +355,11 @@ Use this to adapt your extension to API changes in Sphinx.
 
 .. data:: version_info
```

Comment:
Is there no way to use source code comment? (I'll take a look later)

Reply:
Um, I did add this [here](https://github.com/sphinx-doc/sphinx/pull/4468/files#diff-89b3f0fe820c207f1b0a3e78cd8ad87a).

Reply:
Sorry, what I meant is the way to import the comment using autodoc.
I guess `autodata` or `autoattribute` work well for this.

Reply:
Oh, whoops. I thought I'd done this but I clearly hadn't. Fixed in the next version.